py_neuromodulation/ConnectivityDecoding/_get_grid_whole_brain.py

```diff
@@ -74,22 +71,22 @@ def select_non_zero_voxels(
             coord_.append(mni_coordinates[i, :])
 
         # get only voxel values non-zero
-        ival = np.array(ival)
-        coord_ = np.array(coord_)
-        ival_non_zero = ival[ival != 0]
-        coord_non_zero = coord_[ival != 0]
+        ival_arr  = np.array(ival)
+        coord_arr = np.array(coord_)
+        ival_non_zero = ival_arr[ival != 0]
+        coord_non_zero = coord_arr[ival != 0]
         print(coord_non_zero.shape)
 
         return coord_non_zero, ival_non_zero
 
-    def plot_3d_coordinates(self, coord_non_zero: np.array):
+    def plot_3d_coordinates(self, coord_non_zero: np.ndarray):
         fig = plt.figure()
         ax = fig.add_subplot(111, projection="3d")
         ax.scatter(
             coord_non_zero[:, 0],
             coord_non_zero[:, 1],
             coord_non_zero[:, 2],
-            s=50,
+            s=50, # Bug? Third and fourth argument are s
```

Comment:
That's a 3D plot, so `s` is just the scatter point s.

Reply:
Interesting, in the method definition it looked like it did not accept a third axis and Mypy was complaining, it goes like `def scatter(self, x, y, s=None`. The 3d axes class is actually defined in `mpl_toolkits.mplot3d.Axes3D`, the `projection='3d'` thing must be returning that class instead of the normal axis. Will leave it be then.